src/Kris/LaravelFormBuilder/NewAPI.php

```diff
@@ -0,0 +1,125 @@
+<?php
+	
+	
+	/**
+	 * @method FieldOptionSetters text(      string  $field_name, array $options =[] , boolean $modify = false)
+	 * @method FieldOptionSetters textarea(  string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters radio(     string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters email(     string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters hidden(    string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters password(  string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters image(     string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters number(    string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters file(      string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters submit(    string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters url(       string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters tel(       string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters search(    string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters color(     string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters date(      string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters month(     string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters week(      string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters range(     string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method FieldOptionSetters time(      string  $field_name, array $options = [] , boolean $modify = false)
+	 * @method SelectFieldOptionSetters select_(string  $field_name, array $options = [] , boolean $modify = false)
+	 */
+	class NewAPI extends Form
+	{
+		public $fieldName;
+		public function __call($fieldType,$args = [])
+		{
+			$permitted = [
+			    'text','textarea','radio','checkbox','select',
```

Comment:
Why is this missing `form`, `collection`, and several other fields? Those could be also added.
Also, there is no need to hardcode these here, you can get all available from [here](https://github.com/kristijanhusak/laravel-form-builder/blob/master/src/Kris/LaravelFormBuilder/FormHelper.php#L37)